docs/.vuepress/config.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
             link: "/Vabatahtlik/",
           },
           { text: "Kontakt", link: "/Kontakt/" },
+          { text: "Blogi", link: "/blog/" },
```

Comment:
Other links are starting with capital letter

Reply:
changed small letter to capital